test/scenarios/mixed-reality/input/preview/2020-04-06-preview/test-scenarios/testRemoteRendering.yaml

```diff
@@ -0,0 +1,24 @@
+scope: ResourceGroup
+
+testScenarios:
+  - description: FakedRemoteRenderingScenario
+    steps:
+      - exampleFile: ../examples/remote-rendering/Put.json
+        step: put remote-rendering
+      - exampleFile: ../examples/remote-rendering/RegenerateKey.json
+        step: regenerate-key spatial-anchors
+      - exampleFile: ../examples/remote-rendering/Get.json
+        step: get remote-rendering
+      - exampleFile: ../examples/remote-rendering/GetByResourceGroup.json
+        step: GetByResourceGroup remote-rendering
+      - exampleFile: ../examples/remote-rendering/GetBySubscription.json
+        step: GetBySubscription remote-rendering
+      - exampleFile: ../examples/remote-rendering/ListKeys.json
+        step: ListKeys remote-rendering
+      - exampleFile: ../examples/remote-rendering/Patch.json
+        step: patch remote-rendering
+      - exampleFile: ../examples/remote-rendering/Delete.json
+        step: delete remote-rendering
+
```

Comment:
Just out of curious, what should customers do if they want to add some self-defined scenario and in the meanwhile reserve the original auto-generated one ?

Reply:
Once swagger scenario is configured, we don't generate/modify old test scenario. 
I think the end user can maintain original generated like his/her manual test script if it's useful

Reply:
Don't think this is a good idea. It will impact those RPs who have already onboard to CLI with codegen and they purely want to add a new scenario not entirely replace it. 
It's a breaking change for them if we remove the auto-generated test as long as there's customize test configuration. 
Consider to add a option for it. and we should still generate the auto test by default.

Reply:
My thought is swagger test scenario should take priority than original solution, since swagger scenario is verified and trustable while the original solution is prone to error.
Another solution is we should not clear tests folder when running codegen.

Hi @kairu-ms , feel free to share your preference on this.

Reply:
Added a parameter to allow end user disable swagger test-scenario in latest commit.

And I found a normal second round of codegen will not clear the tests folder, so the original test can still be there and keep no change when swagger test enabled